src/ostorlab/serve_app/app.py

```diff
@@ -27,14 +28,13 @@ def create_app(path: str = "/graphql", **kwargs) -> flask.Flask:
         ),
     )
 
-    @app.route("/")
-    def ui() -> flask.Response:
-        """Serve the Ui folder"""
-        return flask.send_from_directory(str(UI_STATIC_FILES_DIRECTORY), "index.html")
-
-    @app.route("/<path:file_path>")
-    def serve_static(file_path: str) -> flask.Response:
+    @app.route('/')
+    @app.route('/<path:file_path>')
+    def serve_static(file_path="index.html") -> flask.Response:
         """Serve the static files"""
+        full_path = UI_STATIC_FILES_DIRECTORY / file_path
```

Comment:
Are you sure this is not vulnerable to path traversal?

Reply:
@3asm 
I am not using `full_path` to read anything, just for the check.
send_from_directory is used to serve the files and it not vulnurable.